samples/nrf9160/gps/README.rst

```diff
@@ -0,0 +1,126 @@
+.. _gps_with_supl_support_sample:
+
+nRF9160: GPS sockets and SUPL client library
+############################################
+
+The GPS socket sample reads the `GPS`_ data from the `GNSS`_ socket and parses the received frames to interpret the frame contents.
+The GPS socket sample can also download the `A-GPS`_ data from a Secure User-Plane Location (SUPL) server using the SUPL client library, if the library is installed and enabled.
```

Comment:
```suggestion
The GPS socket sample can also download the `A-GPS`_ data from a Secure User-Plane Location (SUPL) server using the :ref:`supl_client` library, if the library is installed and enabled.
```

Reply:
Look at my update, it's not quite like this.